lib/clients/BaseClient.py

```diff
@@ -445,7 +445,7 @@ def format_device(self, device):
         """
         return self.shell('mkfs.ext4 {}'.format(device))
```

Comment:
Do you need to change it here also?

Reply:
Yes..As this has been used for Openstack specific backup, we have to change it here as well.
Will update the PR with the change.